index.js

```diff
@@ -51,6 +52,9 @@ module.exports = function (options) {
       case 'create':
       case 'delete':
         files.push(file.s3.path);
+        if (options.indexRootPath && /index\.html$/.test(file.s3.path)) {
+          files.push(file.s3.path.replace(/index\.html$/, ''))
+        }
```

Comment:
Here is the point of this PR, testing if the S3 path ends with `index.html`, if so, we push another file that just replaces the `index.html` with nothing.